metrics/ci_integrations/lib/data/deserializer/build_data_deserializer.dart

```diff
@@ -19,9 +20,9 @@ class BuildDataDeserializer {
     return BuildData(
       id: id,
       duration: Duration(milliseconds: durationMilliseconds),
-      coverage:
-          json['coverage'] == null ? null : Percent(json['coverage'] as double),
+      coverage: coveragePercent == null ? null : Percent(coveragePercent),
       startedAt: json['startedAt'] as DateTime,
+      apiUrl: json['apiUrl'] as String,
```

Comment:
Should we also add `apiUrl` parsing to a similar deserializer in the web application? https://github.com/platform-platform/monorepo/blob/master/metrics/web/lib/dashboard/data/deserializer/build_data_deserializer.dart